recipes/ogb/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "ogb" %}
+{% set version = "1.2.2" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/ogb-{{ version }}.tar.gz
+  sha256: fb2c67665489477e8ec5b1f858393d724dc9185c3e568c4bc042993d4c18b679
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - outdated
```

Comment:
Is this really needed in host?

Reply:
`grayskull` added it. I'll try to remove it.

Reply:
grayskull was right:

```
Created temporary directory: /tmp/pip-req-tracker-uowwk7oi
Initialized build tracking at /tmp/pip-req-tracker-uowwk7oi
Created build tracker: /tmp/pip-req-tracker-uowwk7oi
Entered build tracker: /tmp/pip-req-tracker-uowwk7oi
Created temporary directory: /tmp/pip-install-l5pbrj2l
Processing $SRC_DIR
  Created temporary directory: /tmp/pip-req-build-potckvou
  Added file://$SRC_DIR to build tracker '/tmp/pip-req-tracker-uowwk7oi'
    Running setup.py (path:/tmp/pip-req-build-potckvou/setup.py) egg_info for package from file://$SRC_DIR
    Created temporary directory: /tmp/pip-pip-egg-info-wfqm44g7
    Running command python setup.py egg_info
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-req-build-potckvou/setup.py", line 9, in <module>
        from version import __version__
      File "/tmp/pip-req-build-potckvou/ogb/version.py", line 6, in <module>
        from outdated import check_outdated  # noqa
    ModuleNotFoundError: No module named 'outdated'
```

will add it back.